android/kotlin/v2/AmpliApp/app/src/main/java/com/amplitude/ampli/Ampli.kt

```diff
@@ -0,0 +1,811 @@
+//
+// Ampli - A strong typed wrapper for your Analytics
+//
+// This file is generated by Amplitude.
+// To update run 'ampli pull kotlin-ampli-v2'
+//
+// Required dependencies: com.amplitude:analytics-android:0.1.0-beta.6
+// Tracking Plan Version: 0
+// Build: 1.0.0
+// Runtime: android:kotlin-ampli-v2
+//
+// [View Tracking Plan](https://data.amplitude.com/test-codegen/Test%20Codegen/events/main/latest)
+//
+// [Full Setup Instructions](https://data.amplitude.com/test-codegen/Test%20Codegen/implementation/kotlin-ampli-v2)
+//
+
+package com.amplitude.ampli
+
+import com.amplitude.android.Amplitude
+import com.amplitude.android.Configuration
+import com.amplitude.android.events.BaseEvent
+import com.amplitude.android.events.EventOptions
+import com.amplitude.android.events.Plan
+
+enum class EventType(val value: String) {
+    Identify("\$identify"),
+    GroupIdentify("\$groupidentify"),
+}
+
+class LoadOptions(
+    val environment: Ampli.Environment? = null,
+    val disabled: Boolean? = null,
+    val client: LoadClientOptions? = null
+)
+
+class LoadClientOptions(
```

Comment:
should we consider server zone information in tracking plan? I added a `zone` parameter in the LoadClientOptions

Reply:
@justin-fiedler , do we want to support overwriting the serverZone.

Reply:
I think we should allow setting a `Configuration` rather than individual fields. Currently Ampli v1 has `LoadClientOptions.options` to set client Options we can do the same here. The naming is kinda strange given options vs configuration. Maybe we should call it `LoadClientOptions.config` or `LoadClientOptions.configuration` in v2. WDYT?

cc @bohan-amplitude @qingzhuozhen @dnshi @kevinpagtakhan

Reply:
I used configuration in v2.